packages/client-sdk-nodejs/test/unit/auth-client.test.ts

```diff
@@ -7,137 +7,135 @@ import {
   Permissions,
   TopicRole,
 } from '@gomomento/sdk-core';
-import {auth} from '@gomomento/generated-types/dist/auth';
 import {permissionsFromScope} from '../../src/internal/internal-auth-client';
-import _GenerateApiTokenRequest = auth._GenerateApiTokenRequest;
+import {permission_messages} from '@gomomento/generated-types/dist/permissionmessages';
+import {TemporaryTokenCachePermissions} from '@gomomento/sdk-core/dist/src/auth/tokens/token-scope';
+
+const textEncoder = new TextEncoder();
+function convert(v: string | Uint8Array): Uint8Array {
+  if (typeof v === 'string') {
+    return textEncoder.encode(v);
+  }
+  return v;
+}
```

Comment:
same thing with this function, not very elegant placement but needed to make CacheItemSelectors into Uint8Arrays